test/typescript/editor.tsx

```diff
@@ -44,7 +44,7 @@ class App extends React.Component<{}, AppState> {
         language="javascript"
         theme="vs-dark"
         value={code}
-        options={{ selectOnLineNumbers: true }}
+        optionsWithoutModel={{ selectOnLineNumbers: true }}
```

Comment:
Wouldn't this be a breaking change?

Reply:
yes. I can make it non-breaking if you'd like - basically we can leave the argument as `options` but just not copy the model if it's passed in additional times.